srcpkgs/mumble/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'mumble'
 pkgname=mumble
-version=1.3.0
-revision=5
+version=1.3.1
+revision=0
```

Comment:
the revisions start at 1, not 0

Reply:
My bad i was reverting commits and something borked. I'll fix that right away.